### DISCUSSION TITLE
要想系统里某个字体不被强制替换该怎么写代码

### DISCUSSION THREAD

**Discussion by jefcn** [F9y4ng/GreasyFork-Scripts#111] [问答专区(Question / Answer)]
目前还在用2021.6的版本。系统里FontAwesome这个字体，不想被强制替换，该如何写代码呢？

**Comment by F9y4ng** [comment]
代码对大部分常用的字体图标的默认应用都有做兼容，升级至最新版再试试。

个别站点更改过默认字体名称的不在以上情况内，可以通过元素过滤排除之。
